services/slam/fake/slam_test.go

```diff
@@ -104,6 +140,40 @@ func TestFakeSLAMGetMap(t *testing.T) {
 	})
 }
 
+func TestFakeSLAMGetInternalStateStream(t *testing.T) {
+	slamSvc := &SLAM{Name: "test", logger: golog.NewTestLogger(t)}
+
+	data := getDataFromStream(t, slamSvc.GetInternalStateStream, slamSvc.Name)
+	test.That(t, len(data), test.ShouldBeGreaterThan, 0)
+
+	data2 := getDataFromStream(t, slamSvc.GetInternalStateStream, slamSvc.Name)
+	test.That(t, data, test.ShouldResemble, data2)
+}
+
+func TestFakeSLAMGetPointMapStream(t *testing.T) {
+	slamSvc := &SLAM{Name: "test", logger: golog.NewTestLogger(t)}
+
+	data := getDataFromStream(t, slamSvc.GetPointCloudMapStream, slamSvc.Name)
+	test.That(t, len(data), test.ShouldBeGreaterThan, 0)
+
+	data2 := getDataFromStream(t, slamSvc.GetPointCloudMapStream, slamSvc.Name)
```

Comment:
add `test.That(t, len(data), test.ShouldBeGreaterThan, 0)` test so that it's more obvious if it fails that it was an issue of lack of data with this call and not unequal data